tools/lib/css_parser.py

```diff
@@ -71,7 +71,7 @@ def is_fluff_token(token):
 
 
 def parse_sections(tokens, start, end):
-    # type: (List[Token], int, int) -> CssSectionList
+    # type: (List[Token], int, int) -> 'CssSectionList'
```

Comment:
What's the story with that quote syntax?  I'm not familiar with it.

Reply:
https://mypy.readthedocs.io/en/latest/kinds_of_types.html#class-name-forward-references